src/types/drbTypes.ts

```diff
@@ -9,12 +9,40 @@ export interface DRBQueryParams {
 
 export type DRBFilters = string[]
 
+export interface DRBResults {
+  works?: DRBWork[]
```

Comment:
I feel like having the types defined here makes the difference explicit (works is an array of DRBWorks while totalWorks is a number). Happy to add a comment though if you think it could be confusing.

Reply:
Yeah, the types does help a lot. But it does mean I have to navigate to this file to figure it out. What if `totalWorks` is renamed `totalWorksNum`? Slightly more explicit.